content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: Device Manager graduates to GA'
+date: 2022-12-19
```

Comment:
The path `content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md` doesn't match the date here.

It'd be nice to fix that. The correct publication date is 2022-12-19.

Reply:
Good catch, Thanks!